TAF/utils/scripts/manual/api-gateway-token.sh

```diff
@@ -0,0 +1,33 @@
+# This script will generate X-Auth-Token from Swagger login
```

Comment:
@bhavikkothari-eaton We are not opening swagger UI page, just suing through curl command through command line.
Should we still use "Swagger login" in comment?

Reply:
We are not using "Swagger login" directly. I will change it as "This script will generate X-Auth-Token".